README.md

```diff
@@ -1,7 +1,9 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S1-CS2103T-F11-4/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-F11-4/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
+**SociaLite** is a project developed for the CS2103T Software Engineering module by [our team](docs/AboutUs.md), based off [AddressBook Level 3](https://github.com/se-edu/addressbook-level3).
+
 * This is **a sample project for Software Engineering (SE) students**.<br>
```

Comment:
Not sure if I should remove this paragraph: is it possibly still relevant for context? I can reword it more to focus on our application instead off Address Book though.

Reply:
Yep the current paragraph sounds slightly off. Perhaps we can remove the paragraph for the time being and add it back once we finish our preliminary features?